src/Components/ExpandableAWS/index.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { ExpandableSection } from '@patternfly/react-core';
 import DescriptionListAWS from '../DescriptionListAWS';
 export const ExpandableAWS = ({ imageName }) => {
-  const [isExpanded, setIsExpanded] = React.useState(false);
+  const [isExpanded, setIsExpanded] = React.useState(true);
```

Comment:
Does that mean that AWS default is open?

Reply:
Yes, is that an issue? :thinking: I thought it is better for the time being (until we have other providers)